pkgs/development/web/ihp-new/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, git, makeWrapper }:
+stdenv.mkDerivation rec {
+  pname = "ihp-new";
+  version = "0.20.0";
+
+  src = fetchFromGitHub {
+    owner = "digitallyinduced";
+    repo = "ihp";
+    rev = "b7f856d982d7e82390f79d109c1f58ef2a365886";
+    sha256 = "sha256-jcuY1bZfpLpJYhh3UQCBEF856O8AT/R6hQjy2cYJhEw";
+  };
+
+  dontConfigure = true;
+  sourceRoot = "source/ProjectGenerator";
+  buildInputs = [ git makeWrapper ];
+  installPhase = ''
+    mkdir -p $out/bin;
+    cp bin/ihp-new $out;
+    makeWrapper $out/ihp-new $out/bin/ihp-new --prefix PATH ":" "${git}/bin";
+  '';
+
+  meta = with lib; {
+    description = "Project generator for the IHP web framework";
```

Comment:
Minor question, feel free to resolve without change: should we expand the `IHP` acronym here to `Integrated Haskell Platform`?

Reply:
good point. Will update this 👍

Reply:
Changed it to `Project generator for the IHP (Integrated Haskell Platform) web framework` 👍